assets/js/modules/idea-hub/datastore/draft-ideas.test.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * `modules/idea-hub` data store: draft-ideas tests.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import { STORE_NAME } from './constants';
+import {
+	createTestRegistry,
+	unsubscribeFromAll,
+} from '../../../../../tests/js/utils';
+import * as fixtures from './__fixtures__';
+import { enabledFeatures } from '../../../features';
+
+describe( 'modules/idea-hub draft-ideas', () => {
+	let registry;
+
+	beforeAll( () => {
+		API.setUsingCache( false );
+	} );
+
+	beforeEach( () => {
+		enabledFeatures.add( 'ideaHubModule' );
+		registry = createTestRegistry();
+	} );
+
+	afterEach( () => {
+		unsubscribeFromAll( registry );
+	} );
+
+	afterAll( () => {
+		API.setUsingCache( true );
+	} );
+
+	describe( 'actions', () => {
+		describe( 'createIdeaDraftPost', () => {
```

Comment:
Can you add a test for the error state as well? Something like https://github.com/google/site-kit-wp/blob/81130720be9540c4c6e9d60e348c4409654e2016/assets/js/googlesitekit/data/create-fetch-store.test.js#L208-L225

Reply:
Added in https://github.com/google/site-kit-wp/pull/3369/commits/4dc1379b59cd3f1e48a4faf92f1bc629ae5a12d4